.github/workflows/build_pex.yml

```diff
@@ -0,0 +1,46 @@
+name: Build PEX file
+
+on:
+  workflow_call:
+    inputs:
+      whl-file-name:
+        required: true
+        type: string
+    outputs:
+      pex-file-name:
+        description: "PEX file name"
+        value: ${{ jobs.build_pex.outputs.pex-file-name }}
+
+jobs:
+  build_pex:
+    name: Build PEX
+    runs-on: ubuntu-latest
+    outputs:
+      pex-file-name: ${{ steps.get-pex-filename.outputs.pex-file-name }}
+    steps:
+    - uses: actions/checkout@v2
```

Comment:
Without parameters, does this just pull the current repo?

Reply:
Yes, current repo, at current ref (in the case of a PR, this defaults to the merge commit of the branch into the target branch).